README.md

```diff
@@ -1,73 +1,214 @@
+
+## contributors
+- Ruthie Rabinovitch | [GitHub](https://github.com/rrabinovitch) | [LinkedIn](https://www.linkedin.com/in/ruthie-r/)
+- Gaby Méndez | [GitHub](https://github.com/gabichuelas) | [LinkedIn](https://www.linkedin.com/in/gabymendez/)
+
+## Setup
+- If you do not yet have `pyenv`, Python3, or Pip3 installed, follow [these](https://opensource.com/article/19/5/python-3-default-mac#what-to-do) instructions.
 - Clone repo: `git clone git@github.com:HireUp-Turing/HireUp_backend.git`
 - Virtual Environment Setup:
   - Build and activate a virtual environment to install your Python packages with `$ python3 -m venv ./venv`
-  - _If you have Python3 set as your global version inside `pyenv` you probably can run `python` instead of `python3` at the beginning of that command.
-  - Activate the virtual environment: `$ source venv/bin/activate`
-  - Run `$ deactivate` to deactivate virtual environment.
+    - If you have Python3 set as your global version inside `pyenv` you probably can run `python` instead of `python3` at the beginning of that command.
+  - Activate the virtual environment: `$ source venv/bin/activate` (Run `$ deactivate` to deactivate the virtual environment when done working with the app)
 - Install Python packages: `$ pip install -r requirements.txt`
-- `$ python run.py` to run server on `localhost:5000`
-
-**Database Setup**
-- `$ createdb hireup_dev` & `$ createdb hireup_test`
-- `$ export DATABASE_URL=postgresql://localhost:5432/hireup_dev`
-- `$ python manage.py db migrate`
-- `$ python manage.py db upgrade`
-- `$ export DATABASE_URL=postgresql://localhost:5432/hireup_test`
-- `$ python manage.py db migrate`
-- `$ python manage.py db upgrade`
+- Set up local databases
+  ```shell
+  $ createdb hireup_dev # creates your dev database
+  $ createdb hireup_test # creates your test database
+  $ export DATABASE_URL=postgresql://localhost:5432/hireup_dev # connects you to your dev database in order to run the following commands
+  $ python manage.py db migrate # add explanation here
+  $ python manage.py db upgrade # runs migrations on your dev database
+  $ python manage.py db_seed # seed data in dev database
+  $ export DATABASE_URL=postgresql://localhost:5432/hireup_test # connects you to your test database in order to run the following commands
+  $ python manage.py db migrate # add explanation here
+  $ python manage.py db upgrade # runs migrations on your dev database
+  ```
+  <!-- do the commands for creating the local db's work without entering the psql console? -->
+- `$ python run.py` to run server on `localhost:5000` (_If you get errors concerning the `FLASK_APP` environment not being set, try `$ export FLASK_APP=manage.py`_)
 
-_If you get errors concerning the `FLASK_APP` environment not being set, try `$ export FLASK_APP=manage.py`_
+<!-- do we still need this part? -->
+<!-- _These instructions will be modified once an `.env` file is added to this repo._ -->
```

Comment:
Commented this part out since we removed the `.env` file. Are we good to remove this?

Reply:
Yeah. Doesn't look like we'll use an .env file anymore